egs/fisher_callhome_spanish/mt1/run.sh

```diff
@@ -184,13 +205,29 @@ if [ ${stage} -le 4 ] && [ ${stop_stage} -ge 4 ]; then
         --seed ${seed} \
         --verbose ${verbose} \
         --resume ${resume} \
-        --train-json ${feat_tr_dir}/data.${src_case}_${tgt_case}.json \
-        --valid-json ${feat_dt_dir}/data.${src_case}_${tgt_case}.json
+        --train-json ${feat_tr_dir}/data_${bpemode}${nbpe}.${src_case}_${tgt_case}.json \
+        --valid-json ${feat_dt_dir}/data_${bpemode}${nbpe}.${src_case}_${tgt_case}.json
 fi
 
 if [ ${stage} -le 5 ] && [ ${stop_stage} -ge 5 ]; then
     echo "stage 5: Decoding"
-    nj=16
+    if [[ $(get_yaml.py ${train_config} model-module) = *transformer* ]]; then
+        # Average NMT models
+        if ${use_valbest_average}; then
+            trans_model=model.val${n_average}.avg.best
+            opt="--log ${expdir}/results/log"
+        else
+            trans_model=model.last${n_average}.avg.best
+            opt="--log"
+        fi
+        average_checkpoints.py \
+            ${opt} \
+            --backend ${backend} \
+            --snapshots ${expdir}/results/snapshot.ep.* \
+            --out ${expdir}/results/${trans_model} \
+            --num ${n_average}
+    fi
+    nj=4
```

Comment:
In the recent ASR and TTS, we move this kind of `nj` before `parse_options` because people with more CPUs can decode faster.

Reply:
Where can I find it?

Reply:
https://github.com/espnet/espnet/blob/2b9e8476b357eee78c0d5896833eeaeb0d1d5efe/egs/ljspeech/tts1/run.sh#L15